puterbot/strategies/summary_mixin.py

```diff
@@ -0,0 +1,105 @@
+"""
+Implements a ReplayStrategy mixin which identifies whether windows are different
```

Comment:
@dianzrong while summarization may be useful, I don't think we will be using it for identifying whether windows are different for the reasons we previously discussed. We may still use it for general purpose summarization, however.

Can you please update this documentation?